include/cuco/detail/common_kernels.cuh

```diff
@@ -261,6 +261,50 @@ __global__ void size(StorageRef storage, Predicate is_filled, AtomicT* count)
   if (threadIdx.x == 0) { count->fetch_add(block_count, cuda::std::memory_order_relaxed); }
 }
 
+template <int32_t BlockSize, typename StorageRef, typename ContainerRef, typename Predicate>
+__global__ void rehash(StorageRef storage_ref, ContainerRef container_ref, Predicate is_filled)
```

Comment:
```suggestion
template <int32_t BlockSize, typename ContainerRef, typename Predicate>
__global__ void rehash(ContainerRef container_ref, Predicate is_filled)
```
`storage_ref` is included in `container_ref` thus no need to duplicate

Reply:
The old `storage_ref` is a different object than the one the `container_ref` holds. So we need to pass it as a separate argument. I agree that the type of the old storage must match the type of the new storage, so we can get rid of the additional type parameter.

Reply:
Addressed in [8262e19](https://github.com/NVIDIA/cuCollections/pull/380/commits/8262e1929673560dcad2c9923a3e11a574fa9931)